<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QPushButton#evilButton {
     background-color: red;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

 QTabWidget::pane { /* The tab widget frame */
     border-top: 2px solid #C2C7CB;
 }
 </string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion enfants</string>
     </attribute>
     <widget class="QPushButton" name="Trier_Enfant">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>460</y>
        <width>84</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  QPushButton {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }</string>
      </property>
      <property name="text">
       <string>TRIER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Recherche_Enfant">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>460</y>
        <width>84</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  QPushButton {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }</string>
      </property>
      <property name="text">
       <string>CHERCHER</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>410</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }</string>
      </property>
      <property name="title">
       <string>Supprimer un enfant :</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_sup">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: darkgray;
     selection-background-color: darkgray;
 }</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Supprimer_Enfant">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>171</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">  QPushButton {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }</string>
       </property>
       <property name="text">
        <string>SUPPRIMER</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>311</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  
 QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }</string>
      </property>
      <property name="title">
       <string> Ajouter ou modifier un enfant  :</string>
      </property>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QLineEdit { color: red }</string>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Pr√©nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Niveau </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_idE">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
QLineEdit{ background-color:rgb(202, 255, 227);
border: 1px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date de naissance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nomE">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: darkgray;
     selection-background-color: darkgray;
 }</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenomE">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: darkgray;
     selection-background-color: darkgray;
 }</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Remarque</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_remq">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: darkgray;
     selection-background-color: darkgray;
 }</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sexe</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="la_date">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_enfant">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>340</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">  QPushButton {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }</string>
       </property>
       <property name="text">
        <string>AJOUTER</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Modifier_Enfant">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>340</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">  QPushButton {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }</string>
       </property>
       <property name="text">
        <string>MODIFIER</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Photo d'identit√©</string>
       </property>
      </widget>
      <widget class="QLabel" name="le_pic">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Gar√ßcon </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fille</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>180</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>P</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QTableView" name="tabEnfant">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>441</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QTableView {
        selection-background-color: blue;
        background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
        font: 1em;
        color: white;
       border: 2px solid gray;
       border-radius: 5px;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Iprim">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Gestion parents</string>
     </attribute>
     <widget class="QTableView" name="tabParent">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>431</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QTableView {
        selection-background-color: blue;
        background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
        font: 1em;
        color: white;
       border: 2px solid gray;
       border-radius: 5px;
}</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>211</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }
</string>
      </property>
      <property name="title">
       <string>Supprimer parent/s par Id </string>
      </property>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Id :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_idp_sup">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Sup_p">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  
 QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }</string>
      </property>
      <property name="title">
       <string>Ajouter ou Modifier un parent </string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Idp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom_p</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pr√©nom_p</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Num√©ro de telephone</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Relation</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_idP">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: darkgray;
     selection-background-color: darkgray;
 }</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nomP">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenomP">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_add">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="OK">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>180</y>
         <width>107</width>
         <height>20</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radM">
          <property name="text">
           <string>M√®re</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radP">
          <property name="text">
           <string>P√®re</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="Ajouter_Parent">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>84</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">  QPushButton {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Modif_p">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modif</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Recherche_p">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
